res/css/views/elements/_RichText.scss

```diff
@@ -13,7 +13,7 @@
     height: $font-16px;
     line-height: $font-20px;
     padding-left: 0;
-    padding-right: 0.5em;
+    padding-right: 0.5rem;
```

Comment:
I assume `em` and `rem` are equivalent here?

Reply:
Turns out they're not. But the difference is very subtle. A few commits later I use $font-5px. I've reverted it to 0.4ems.